### PR TITLE
fix: Orchestrator bugs

### PR DESCRIPTION
**Pull request by TBonnin**
a couple of small bugfixes for the orchestrator. Each commit should be self-explanatory

### CODE REVIEW

File: packages/scheduler/lib/scheduler.ts [hunk 1]
```diff
@@ -330,18 +330,24 @@ export class Scheduler {
...
-            if (!schedule.value[0]) {
+            if (!found.value[0]) {
                 return Err(`Schedule '${scheduleName}' not found`);
             }
+            const schedule = found.value[0];
+
+            if (schedule.state === state) {
+                // No-op if the schedule is already in the desired state
+                return Ok(schedule);
+            }
```
Comment (TBonnin) [inline packages/scheduler/lib/scheduler.ts:347]: making `setScheduleState` a no-op if newState == currentState

File: packages/scheduler/lib/workers/scheduling/scheduling.worker.ts [hunk 2]
```diff
@@ -114,9 +114,9 @@ export class SchedulingChild {
                 });
                 if (task.isErr()) {
                     logger.error(`Failed to create task for schedule: ${schedule.id}`);
-                    return;
```
Comment (TBonnin) [inline packages/scheduler/lib/workers/scheduling/scheduling.worker.ts:117]: continuing  on task creation failure

File: packages/orchestrator/lib/clients/validate.ts [hunk 3]
```diff
@@ -133,7 +133,7 @@ export function validateTask(task: Task): Result<OrchestratorTask> {
             })
         );
     }
-    return Err(`Cannot validate task ${JSON.stringify(task)}: ${action.error || webhook.error || postConnection.error}`);
+    return Err(`Cannot validate task ${JSON.stringify(task)}: ${sync.error || action.error || webhook.error || postConnection.error}`);
```
Comment (bodinsamuel) [inline packages/orchestrator/lib/clients/validate.ts]: none of there are stringified? :D

File: packages/orchestrator/lib/events.ts [hunk 4]
```diff
@@ -23,10 +23,11 @@ export class EventsHandler extends EventEmitter {
         this.onCallbacks = {
             CREATED: (task: Task) => {
                 on.CREATED(task);
-                this.emit(`task:started:${task.groupKey}`, task);
+                this.emit(`task:created:${task.groupKey}`, task);
```
Comment (bodinsamuel) [inline packages/orchestrator/lib/events.ts:26]: oops, after 15 years I still can't understand how we can read the same lines over and over, and still miss stuff like this ahah Brain is goofy

**Review by bodinsamuel — APPROVED** [review]

**Pull request merged**